tsl/test/expected/dist_am-12.out

```diff
@@ -0,0 +1,120 @@
+DROP DATABASE IF EXISTS data_node_3;
+SET client_min_messages TO NOTICE;
+-- Add data nodes using the TimescaleDB node management API
+SELECT * FROM add_data_node('data_node_1', host => 'localhost',
+                            database => 'data_node_1');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_1 | localhost | 55432 | data_node_1 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_2', host => 'localhost',
+                            database => 'data_node_2');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_2 | localhost | 55432 | data_node_2 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_3', host => 'localhost',
+                            database => 'data_node_3');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_3 | localhost | 55432 | data_node_3 | t            | t                | t
+(1 row)
+
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO PUBLIC;
+-- Create a new table access method by reusing heap handler
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler;
+SELECT * FROM test.remote_exec('{ data_node_1, data_node_2, data_node_3 }', $$
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler;
+$$);
+NOTICE:  [data_node_1]: 
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler
+NOTICE:  [data_node_2]: 
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler
+NOTICE:  [data_node_3]: 
+CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler
+ remote_exec 
+-------------
+ 
+(1 row)
+
+-- Create distributed hypertable using non-default access method
+CREATE TABLE disttable(time timestamptz NOT NULL, device int) USING test_am;
+SELECT * FROM create_distributed_hypertable('disttable', 'time', 'device', 3);
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | disttable  | t
+(1 row)
+
+-- Make sure that distributed hypertable created on data nodes is
+-- using the correct table access method
+SELECT * FROM test.remote_exec('{ data_node_1, data_node_2, data_node_3 }', $$
+SELECT cl.relname FROM pg_am am, pg_class cl WHERE cl.relam = am.oid and am.amname = 'test_am';
+$$);
+NOTICE:  [data_node_1]: 
+SELECT cl.relname FROM pg_am am, pg_class cl WHERE cl.relam = am.oid and am.amname = 'test_am'
+NOTICE:  [data_node_1]:
+relname  
+---------
+disttable
```

Comment:
Nit: For me it would be more intuitive to return the amname here and have the `'disttable'::regclass` in the WHERE clause.